frontend/src/features/map_planning/layers/plant/hooks/useFindSeedById.ts

```diff
@@ -0,0 +1,28 @@
+import { findSeedById } from '@/features/seeds/api/findSeedById';
+import { useQuery } from '@tanstack/react-query';
+import { useTranslation } from 'react-i18next';
+import { toast } from 'react-toastify';
+
+/**
+ * Load a seed from the backend using its id.
+ *
+ * @param seedId the id of the requested seed.
+ * @param enabled will disable the backend query if set to false.
+ * @param quiet will disable error modals if set to true.
+ */
+export function useFindSeedById(seedId: number, enabled = true, quiet = false) {
+  const { t } = useTranslation(['seeds']);
+  const { data, error } = useQuery(['plants/seed', seedId] as const, {
+    queryFn: (context) => findSeedById(context.queryKey[1]),
+    enabled,
+    staleTime: Infinity,
+  });
+
+  if (error && !quiet) {
+    toast.error(t('seeds:error_fetching_seed'), { autoClose: false });
```

Comment:
Why is autoClose false here?

Reply:
This code was not originally added by me.
As far as I can tell the reasoning is that the user does not miss an error message because it is not displayed long enough.

Reply:
I don't think this should be randomly chosen. @tschawax what do you think is suitable as UI guideline?